packages.yml

```
@@ -1,3 +1,3 @@
 packages:
-  - git: https://github.com/fivetran/dbt_greenhouse_source.git
-    warn-unpinned: false # todo: change to dbt hub before release
\ No newline at end of file
+  - package: fivetran/greenhouse_source
+    version: 0.1.0
```

can you do the range here instead?
```suggestion
    version: [">=0.1.0","<0.2.0"]
```